app/components/verkiezingen/prepare-legislatuur-section.js

```diff
@@ -265,14 +268,15 @@ export default class PrepareLegislatuurSectionComponent extends Component {
     this.isGeneratingRows = false;
   }
 
-  onCreate = restartableTask(async ({ instanceId }) => {
+  @action
+  async onCreate({ instanceId }) {
     this.editMode = null;
     const mandataris = await this.store.findRecord('mandataris', instanceId);
```

Comment:
weirdly, this function stopped after this promise (without exception) if it was a restartable task for me. changing it to action worked.